### PR TITLE
Bug 1882660: Show all operators that will be approved with manual ins…

### PR DESCRIPTION
**Pull request by rhamilto**
…tall plan

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1882660

Screen Shot 2020-11-02 at 1 35 51 PM
Screen Shot 2020-11-02 at 3 38 15 PM

### CODE REVIEW

**Comment by itsptk** [comment]
LGTM 👍

File: frontend/packages/operator-lifecycle-manager/src/components/index.tsx [hunk 1]
```diff
@@ -147,11 +149,32 @@ export const exampleForModel = (csv: ClusterServiceVersionKind, model: K8sKind)
     ),
   );
 
+const InstallPlanCSVNames: React.FC<InstallPlanReviewProps> = ({ installPlan }) =>
+  installPlan?.spec.clusterServiceVersionNames
+    .sort()
+    .map((CSVName, i) => <strong key={`${CSVName}-${i}`}>{CSVName}</strong>)
+    .map((CSVName, i) => (i > 0 ? <span key={`${CSVName}-${i}`}>, {CSVName}</span> : CSVName));
```
[the remark this answers is not in the document]
Comment (rhamilto) [reply frontend/packages/operator-lifecycle-manager/src/components/index.tsx:156]: Yeah.  The first one adds the `<strong>` and the second one adds the `, ` if not the first instance.  Perhaps there is a more elegant way to do this?

[the remark this answers is not in the document]
Comment (rhamilto) [reply frontend/packages/operator-lifecycle-manager/src/components/index.tsx:205]: ^ @spadgett

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**